sdk/core/Azure.Core/api/Azure.Core.net6.0.cs

```diff
@@ -1150,8 +1158,8 @@ public partial class ModelJsonConverter : System.Text.Json.Serialization.JsonCon
     }
     public static partial class ModelSerializer
     {
-        public static object Deserialize(System.BinaryData data, System.Type returnType, Azure.Core.Serialization.ModelSerializerFormat format) { throw null; }
-        public static object Deserialize(System.BinaryData data, System.Type returnType, Azure.Core.Serialization.ModelSerializerOptions? options = null) { throw null; }
+        public static object Deserialize(System.BinaryData data, [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.NonPublicConstructors | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)] System.Type returnType, Azure.Core.Serialization.ModelSerializerFormat format) { throw null; }
+        public static object Deserialize(System.BinaryData data, [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembersAttribute(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.NonPublicConstructors | System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicConstructors)] System.Type returnType, Azure.Core.Serialization.ModelSerializerOptions? options = null) { throw null; }
         public static T Deserialize<T>(System.BinaryData data, Azure.Core.Serialization.ModelSerializerFormat format) where T : Azure.Core.Serialization.IModelSerializable<T> { throw null; }
         public static T Deserialize<T>(System.BinaryData data, Azure.Core.Serialization.ModelSerializerOptions? options = null) where T : Azure.Core.Serialization.IModelSerializable<T> { throw null; }
```

Comment:
Do these need to be updated?

Reply:
I assume you are asking about the `Deserialize<T>` variant?  [They are in the code here](https://github.com/Azure/azure-sdk-for-net/pull/38010/files/6759723d164b43786c78b7c47ddf12014b5bb040#diff-3b9bfd9e4c2b395a13060137c8086bb5a76bf09722bf56b49c9059001cd623e6R76), but I think our api exporter might have an issue and isn't showing it on the type parameter.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/38084